cypress/support/fragments/inventory/inventoryActions.js

```diff
@@ -1,5 +1,10 @@
-import { Button } from '../../../../interactors';
+import { Button, Select, TextField, TextInput } from '../../../../interactors';
```

Comment:
Can you fix github comments please?
![image](https://user-images.githubusercontent.com/26041116/144597728-2a825ac7-37c8-45eb-9e5a-422f8d4d480a.png)